### PR TITLE
cleaned up integration-cli/docker_api_inspect_unix_test.go

### PR DESCRIPTION
**Pull request by ZJaffee**
As Part of "Proposal: use of checkers on Integration test #16756"

### CODE REVIEW

**Comment by vdemeester** [comment]
As stated by the build you need to format the modified files, with `gofmt -s -w`

**Comment by HuKeping** [comment]
ping @ZJaffee

**Comment by ZJaffee** [comment]
Sorry I'll get these done later today, I was out of town at a conference last week.

File: integration-cli/docker_api_inspect_unix_test.go [hunk 1]
```diff
@@ -19,20 +20,17 @@ func (s *DockerSuite) TestInspectApiCpusetInConfigPre120(c *check.C) {
...
-	c.Assert(status, check.Equals, http.StatusOK)
-	c.Assert(err, check.IsNil)
+	c.Assert(status, checker.Equals, http.StatusOK)
+	c.Assert(err, checker.IsNil)
 
 	var inspectJSON map[string]interface{}
-	if err = json.Unmarshal(body, &inspectJSON); err != nil {
-		c.Fatalf("unable to unmarshal body for version 1.19: %v", err)
-	}
+	c.Assert(json.Unmarshal(body, &inspectJSON),checker.IsNil,Commentf("unable to unmarshal body for version 1.19: %v",json.Unmarshal(body, &inspectJSON)))
```
Comment (cpuguy83) [inline integration-cli/docker_api_inspect_unix_test.go]: How come this is calling Unmarshal a 2nd time?
If it's already error'd out from unmarshalling, it probably won't work a 2nd time like this.

**Comment by calavera** [comment]
I'm very sorry but I'm going to close this PR until the errors are resolved. Please, feel free to send it again after testing that every test still works and the code is valid. `make test` and `make validate` are your friends.

As a rule of thumb, before opening all your PRs again, it will be easier to get them merged if you focus in one at a time, that way reviewers don't need to point the same errors over several changes.

**Pull request closed without merging**